### PR TITLE
CYBL-989-Add-Quota-Support-On-v3

### PR DESCRIPTION
**Pull request by ahmadiesa-abu**
Added Openstack Project quota operations

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     license='LICENSE',
     zip_safe=False,
     packages=find_packages(exclude=['tests*']),
-    install_requires=['cloudify-plugins-common',
+    install_requires=['cloudify-common>=4.4',
```
Comment (mabuaisha) [inline setup.py]: @ahmadiesa-abu I think this added here for a reason, so not sure if we should change this again to common

@EarthmanT @isaac-s

Comment (ahmadiesa-abu) [reply setup.py]: @mabuaisha moving it back , it was merged from another branch by mistake

Comment (0lvin) [reply setup.py]: Change is based on EOL https://cloudify.co/cloudify-lifecycle/ - looks like cloudify-plugins-common should retire. What do you think?

Comment (ahmadiesa-abu) [reply setup.py]: @0lvin we decided to move it back as it should use the cloudify-plugins-common like it was before

**Pull request merged**